src/energyplus/ForwardTranslator.cpp

```diff
@@ -594,6 +594,22 @@ namespace energyplus {
       translateAndMapModelObject(vrf);
     }
 
+    // get AirConditionerVariableRefrigerantFlowFluidTemperatureControl objects in sorted order
+    std::vector<AirConditionerVariableRefrigerantFlowFluidTemperatureControl> vrfftcs =
+      model.getConcreteModelObjects<AirConditionerVariableRefrigerantFlowFluidTemperatureControl>();
+    std::sort(vrfftcs.begin(), vrfftcs.end(), WorkspaceObjectNameLess());
+    for (AirConditionerVariableRefrigerantFlowFluidTemperatureControl vrfftc : vrfftcs) {
```

Comment:
I know you're copying above, but to avoid having to do another pass ocne #4532 is merged, please take a reference here

```suggestion
    for (auto& vrfftc : vrfftcs) {
```